include/uxr/agent/transport/Server.hpp

```diff
@@ -33,9 +33,9 @@ class Server
     Server();
     virtual ~Server();
 
-    UXR_AGENT_DLLAPI bool run(bool discovery_enabled = false);
-    UXR_AGENT_DLLAPI bool stop();
-    UXR_AGENT_DLLAPI bool load_config_file(const std::string& path);
+    UXR_AGENT_LIB bool run(bool discovery_enabled = false);
```

Comment:
I also want to remove the word `DLL` because is too much Windows. But maybe we need to relate the `UXR_AGENT_LIB` with some that implies *exporting*.

Reply:
Ok,  I changed it by `UXR_AGENT_EXPORT`.